validationframework-swing/src/main/java/com/google/code/validationframework/swing/decoration/IconComponentDecoration.java

```diff
@@ -218,6 +238,14 @@ public IconComponentDecoration(JComponent owner, AnchorLink anchorLinkWithOwner,
         decorationPainter.addMouseListener(toolTipVisibilityAdapter);
         decorationPainter.addMouseMotionListener(toolTipVisibilityAdapter);
         decorationPainter.addComponentListener(toolTipVisibilityAdapter);
+
+        scheduleTooltipUpdateTimer = new Timer(MINIMUM_TOOLTIP_VISIBLE_TIME_MS, new ActionListener() {
```

Comment:
We should do some clean-up in the `dispose()` method: at least stop the `scheduleTooltipUpdateTimer`.